versions/1/manifest.spec.yml

```diff
@@ -106,6 +106,21 @@ spec:
         - "1.0.0-SNAPSHOT"
         - "1.0.0-dev.1"
   properties:
+    elasticsearch:
```

Comment:
I noticed that these properties are more or less sorted in prioritized order, which means that "elasticsearch" as extension should be at the bottom.